test/katex-spec.js

```diff
@@ -2975,6 +2975,25 @@ describe("A macro expander", function() {
         }}));
     });
 
+    it("should delay expansion if preceded by \\expandafter", function() {
+        expect`\expandafter\foo\bar`.toParseLike("x+y", new Settings({macros: {
+            "\\foo": "#1+#2",
+            "\\bar": "xy",
+        }}));
+        expect`\expandafter\foo\def\foo{x}`.not.toParse();
+        expect`\def\bar{\def\foo{}}\expandafter\bar\foo`.not.toParse();
```

Comment:
The first one shouldn't parse because there's nothing to expand with `\def`, right?  I'm not sure about why the second one shouldn't parse though.